src/data/faq.json

```diff
@@ -800,71 +800,11 @@
                                 "anchor": "val_service_basics",
                                 "active": false,
                                 "textblock": [
+                                    "<b>UPDATE January 10, 2022</b><br/> The Europe-wide requirements for online verification services have already been in place since October 2021. In Germany, <b>no provider of an online verification service has been approved yet</b>. Tests are currently being carried out, the results of which are being awaited for the time being. Online verification services will not be available in practice until these tests have been evaluated and the Federal Commissioner for Data Protection and Freedom of Information (BfDI) has provided advice. With version 2.15, however, the CWA is ready to support corresponding verification services as soon as providers of such a service are approved.<hr/>",

```

Comment:
```suggestion
                                    "<b>UPDATE  April 12, 2022</b><br/> The Europe-wide requirements for online verification services have already been in place since October 2021. In Germany, <b>no provider of an online verification service has been approved</b>. With version 2.15, however, the CWA is ready to support corresponding verification services.<hr/>",
```

@GisoSchroederSAP Does this sound good to you?